docs/topics/gradle.md

```diff
@@ -316,10 +299,43 @@ kotlin{
 
 </tabs>
 
-> You can use shorthand for a dependency on a Kotlin module, for example, kotlin("test") for "org.jetbrains.kotlin:kotlin-test".
+> You can use shorthand for a dependency on a Kotlin module, for example, kotlin('test') for "org.jetbrains.kotlin:kotlin-test".
 >
 {type="note"}
 
+You can choose JUnit 5 or TestNG by calling 
+[`useJUnitPlatform()`]( https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform) 
+or [`useTestNG()`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useTestNG) in the test task:
+
+ <tabs>
+
+```groovy
+tasks {
+    test {
+        // enable TestNG support
+        useTestNG()
+        // or
+        // enable JUnit Platform (a.k.a. JUnit 5) support
+        useJUnitPlatform()
+    }
+}
+```
+
+```kotlin
+tasks.test {
+    // enable TestNG support
+    useTestNG()
+    // or
+    // enable JUnit Platform (a.k.a. JUnit 5) support
+    useJUnitPlatform()
+}
+```
+
+ </tabs>
+
+You can disable the automatic selection of a testing framework by adding the line `kotlin.test.infer.jvm.variant=false` 
```

Comment:
It's not a testing framework that is selected automatically, but a variant of `kotlin-test` library.